### DISCUSSION TITLE
Multiple cameras

### DISCUSSION THREAD

**Discussion by Adri314** [carla-simulator/carla#5785] [Q&A]
Hi! 
Is there any way to display the simulation using multiple cameras? For example, having a couple of cameras attached to the same player and having the output show in different pygame windows.
Thanks!

**Comment by JosenJiang** [comment]
Try cv2 python module? I use it to show different cameras.

**Reply by Adri314** [reply]
Thanks! I haven't tried that yet. I haven't actually used that module before, so thanks so much for pointing me in that direction.

**Reply by JosenJiang** [reply]
I hope this can help you: https://www.geeksforgeeks.org/python-opencv-cv2-imshow-method/

**Reply by Adri314** [reply]
Thanks so much! I see that cv2 and the imshow method are the approach that is used here: https://github.com/fcagroupj/fcarsim. I will build on this to add cameras.
